requirements/dev.txt

```diff
@@ -70,6 +71,10 @@ coverage[toml]==7.2.1
     #   -r requirements/quality.txt
     #   codecov
     #   pytest-cov
+cryptography==40.0.1
```

Comment:
Not really, no. Ideally there would be an easy way for people on Macs to run `make upgrade` in a Linux environment so that it's consistent with the deployed environments (and the GitHub upgrade-requirements workflow results) but I don't think there was any resolution on that. At this point I just eyeball the changes and recognize which ones are the "usual suspects".